linux_sql/scripts/host_info.sh

```diff
@@ -0,0 +1,60 @@
+#! /bin/bash
+
+#validate number of arguments
+if [ "$#" -ne 5 ]; then
+    echo "Illegal number of parameters"
+    exit 1
+fi
+
+#store arguments in variables
+server_hostname=$1
+port=$2
+db_name=$3
+uname=$4
+password=$5
+
+#validate port number
+if [ $port -ne 5432 ]; then
+	echo "Please enter correct port number"
+	exit 1
+fi
+
+#validate database name
+if [ $db_name != "host_agent" ]; then
+	echo "Please enter correct database name"
+	exit 1
+fi
+
+#validate user name
+if [ $uname != "postgres" ]; then
+        echo "Please enter correct user name"
+        exit 1
+fi
+
+#validate password
+if [ $password != "password" ]; then
+        echo "Please enter correct password"
+        exit 1
+fi
```

Comment:
you do not need to check these parameter values..user can set it to anything they want 

e.g. a user may provision a PSQL with a difference username and password..